### ISSUE TITLE
resetToDefaultPosition: public

### ISSUE THREAD

**Issue by AYastrebov** [shergin/SherginScrollableNavigationBar#2]
Please, make resetToDefaultPosition: method as public

**Comment by alexanderfrey** [comment]
yes, need to have a way to show navigation bar again programatically e.g. when pushing a new view controller or when double tapping on the scrollview...

**Comment by void-main** [comment]
I tried to secretly copy the signature to .h file, and used it in the `viewWillDisappear:` and `viewDidLoad` method when pushing a new view controller, the frame did change to normal, but when the new view controller displayed, the nav bar still locates in the old position, and blinked back to normal.
